CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+### 0.5.1 (unreleased)
+
+- New features
+  - More consistent handling of missing keys: if a config key is missing pureconfig always throws a
+    `KeyNotFoundException` now, unless the `ConfigConvert` extends the new `AllowMissingKey` trait.
+
+### 0.5.0 (Jan 3, 2017)
+
+- New features
+  - Sealed families are now converted to and from configs unambiguously by using an extra `type` field (customizable) in
+    their config representation; 
+  - New `CoproductHint` trait which allows customization of the derived `ConfigConvert` for sealed families;
+- Breaking changes
+  - The default config representation for sealed families has changed. Configs for such classes that were read
```

Comment:
I also would like to add the links, people will probably check the CHANGELOG on github and being able to go to the section of the README describing the new feature can be very useful.